content/news/release-20200202.md

```diff
@@ -0,0 +1,42 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+While we work on the next full playtest (more on this below), there are a few serious issues that we wanted to address without futher delay. [Release 20200202](/download/) is now available, including:
+
+* Fixes for significant performance and behaviour regressions with aircraft
+* Fixes for rare crashes during gameplay and when loading saved games
+* Fixes for several issues with mission scripts and the skirmish AI
+* Fixes for missing team chat in game replays
+* Fixes for build failures caused by an [upstream API change](https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/)
+* Improvments to the build icon layout in RA and D2k sidebars
+* Improvements to the in-game menu option layout
+
+As always, the [full Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/44d23a103e9dc3e06c1944bbf9f9f411f865066f) includes more detail, and we welcome feedback in the comments below, on [our forum](https://forum.openra.net), [Discord](https://discord.openra.net), or [GitHub](https://github.com/OpenRA/OpenRA/) pages.
+
+Progress is converging on our [Next Release](https://github.com/OpenRA/OpenRA/milestone/29) milestone, and we hope to start playtesting in the near future. Some of the changes to look forward to include improvements to production rally points and unit pathfinding, new features for the map editor, and major improvements to the in-game UI.
+
+Members of our Discord and IRC channels already know that the headline feature for the next release are the long-awaited abilities to smoothly zoom the battlefield using the mouse wheel and to adjust the size of the in-game UI. These features are built on top of a much larger rework of the game rendering, which includes dropping support for the legacy OpenGL 2 graphics API. If you can't wait to try these, or would like to help with testing (especially if you use Linux with an AMD graphics card), we encourage you to take a look at [this forum thread]().
```

Comment:
Since it's technically two features (even though related), I'd use plural 'features' in the first line, otherwise it sounds a bit weird with 'are' and 'abilities'.